ragaai_catalyst/tracers/utils/langchain_tracer_extraction_logic_old.py

```
@@ -0,0 +1,50 @@
+                        )["kwargs"]["text"]
+                        result["data"]["context"] = context
+                        result["data"]["system_prompt"] = system_prompt
+                    if (
+                            trace["name"] == "ChatOpenAI.langchain.task"
+                            or trace["name"] == "ChatGroq.langchain.task"
+                    ):
+                        response = trace["attributes"]["gen_ai.completion.0.content"]
+                        result["data"]["response"] = response
+                except Exception as e:
+                    result["Error"] = str(e)
+        for key,value in trace_aggregate.items() :
```

Ruff (0.8.2)

Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)